app/assets/javascripts/kassi.js

```diff
@@ -337,9 +337,8 @@ function updateSellerGetsValue(priceInputSelector, displayTargetSelector, curren
     }
 
     $display.text(
-      [ST.paymentMath.displayMoney(Math.max(0, displaySum)),
-       $currency.val()]
```

Comment:
It's used on line 345. I need to change the positioning of this nearer there to better reflect where and why it is used. I didn't yet touch it as I was concentrating on other things.